src/app/career-table/career-table.component.ts

```diff
@@ -1,45 +1,62 @@
-import { Component, OnInit } from '@angular/core';
-import { DataService } from '../_services/data.service';
+import { Component, OnInit } from "@angular/core";
+import { DataService } from "../_services/data.service";
+import { NgbModal } from "@ng-bootstrap/ng-bootstrap";
+import { RoleInformationComponent } from "../role-information/role-information.component";
+import { Role } from "../_classes/role";
 
 @Component({
-  selector: 'app-career-table',
-  templateUrl: './career-table.component.html',
-  styleUrls: ['./career-table.component.css']
+  selector: "app-career-table",
+  templateUrl: "./career-table.component.html",
+  styleUrls: ["./career-table.component.css"]
 })
 export class CareerTableComponent implements OnInit {
-
-  rolesCapabilityMap = new Map<string, string[]>();
+  rolesCapabilityMap = new Map<string, Role[]>();
   capabilities = [];
   roleCapabilityData: any;
   departmentName: any;
 
-  constructor(private dataService: DataService) { }
- 
+  roles = [];
+
+  constructor(
+    private dataService: DataService,
+    private modalService: NgbModal
+  ) {}
+
   ngOnInit() {
-    this.dataService.getCapabilityNamesByDepartment(1).then(response =>{
```

Comment:
Chat to @sapphireduffy and @jurassic-marc about the changes they've been doing here. They have the logic to pass in the correct department ID (which is here, https://github.com/sapphireduffy/KainosCareerLattice/pull/5). So it'll need merged before you merge this